data/scenarios/README.md

```diff
@@ -0,0 +1,214 @@
+means the key is required; other keys are optional and take on the
+indicated default value when they are not present.
+
+As with all YAML, note that the order of keys in a key-value mapping
+does not matter.
+
+| Key            | Default? | Type       | Description                                                                                                                                                                                                                                                                                                                                                   |
+|----------------|----------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name`         |          | `string`   | The name of the scenario. For official scenarios, this is what shows up in the new game menu.                                                                                                                                                                                                                                                                 |
+| `description`  | `""`     | `string`   | A short description of the scenario. This shows up next to the new game menu when the scenario is selected.                                                                                                                                                                                                                                                   |
+| `creative`     | `False`  | `boolean`  | Whether the scenario should start out in creative mode.                                                                                                                                                                                                                                                                                                       |
+| `seed`         | `null`   | `int`      | An optional seed that will be used to seed the random number generator.  If a procedurally generated world is used, the seed hence determines the world.  Hence, if the seed is specified, the procedurally generated world will be exactly the same every time, for every player.  If omitted, a random seed will be used every time the scenario is loaded. |
+| `entities`     | `[]`     | `map list` | An optional list of custom entities, to be used in addition to the built-in entities.  See [Entities](#entities).                                                                                                                                                                                                                                             |
```

Comment:
The `map list` is not rendered very nicely in README and is IMHO too confusing.

I would suggest using:
```markdown
[`entity`](#entites) list
```